samples/react-sample-app/package.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "identity-verification-sample-app",
+    "version": "1.0.0",
+    "description": "React based sample SPA to demonstrate Identity Verification with WSO2 Identity Server",
+    "keywords": [
+        "Asgardeo",
+        "WSO2",
+        "WSO2 IS",
+        "WSO2 Identity Server",
+        "OIDC",
+        "SDK",
+        "React",
+        "Single Page Application",
+        "SPA",
+        "Sample",
+        "Javascript",
+        "Typescript",
+        "Authenticate",
+        "Identity Verification"
+    ],
+    "author": "WSO2",
+    "license": "Apache-2.0",
+    "private": false,
+    "scripts": {
+        "start": "webpack-dev-server --mode development --hot --open",
+        "prebuild": "yarn install",
+        "build": "webpack"
+    },
+    "dependencies": {
+        "@asgardeo/auth-react": "^2.0.3",
+        "@babel/runtime-corejs3": "^7.20.6",
+        "@mui/icons-material": "^5.13.7",
+        "@mui/material": "^5.13.7",
+        "@onfido/active-video-capture": "^0.28.7",
+        "@onfido/api": "^3.4.0",
+        "@textea/json-viewer": "^2.14.0",
+        "crypto-browserify": "^3.12.0",
+        "onfido-sdk-ui": "^14.34.0",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-router-dom": "^6.10.0",
+        "stream-browserify": "^3.0.0",
+        "util": "^0.12.5"
+    },
+    "devDependencies": {
+        "@babel/cli": "^7.19.3",
+        "@babel/core": "^7.20.5",
+        "@babel/plugin-proposal-class-properties": "^7.18.6",
+        "@babel/plugin-proposal-decorators": "^7.20.5",
+        "@babel/preset-env": "^7.20.2",
+        "@babel/preset-react": "^7.18.6",
+        "@babel/preset-typescript": "^7.18.6",
+        "@types/react": "^18.0.26",
+        "@types/react-dom": "^18.0.9",
+        "@typescript-eslint/eslint-plugin": "^5.61.0",
+        "@typescript-eslint/parser": "^5.61.0",
+        "babel-loader": "^9.1.0",
+        "babel-polyfill": "^6.26.0",
+        "chalk": "^5.2.0",
+        "core-js": "^3.26.1",
+        "css-loader": "^6.7.3",
+        "dev-server-ports": "^2.0.0",
+        "dotenv": "^16.0.3",
+        "eslint": "^8.44.0",
+        "eslint-plugin-react": "^7.32.2",
+        "eslint-plugin-react-hooks": "^4.6.0",
+        "html-loader": "^4.2.0",
+        "html-webpack-plugin": "^5.5.0",
+        "source-map-loader": "^4.0.1",
+        "style-loader": "^3.3.1",
+        "typescript": "^4.9.5",
+        "url-loader": "^4.1.1",
+        "webpack": "^5.75.0",
+        "webpack-cli": "^5.0.1",
+        "webpack-dev-server": "^4.11.1"
+    }
+}
```

Comment:
Let's add the end of file blank lines. Check other places as well

Reply:
Resolved at https://github.com/wso2-extensions/identity-verification-onfido/pull/17